app/app.js

```diff
@@ -1,12 +1,14 @@
 import Application from '@ember/application';
 import Resolver from 'ember-resolver';
 import loadInitializers from 'ember-load-initializers';
-import config from 'css-modules-to-scoped-css/config/environment';
+import config from './config/environment';
+
+import compatModules from '@embroider/virtual/compat-modules';
 
 export default class App extends Application {
   modulePrefix = config.modulePrefix;
   podModulePrefix = config.podModulePrefix;
-  Resolver = Resolver;
+  Resolver = Resolver.withModules(compatModules);
 }
 
-loadInitializers(App, config.modulePrefix);
+loadInitializers(App, config.modulePrefix, compatModules);
```

Comment:
Most of the changes in this PR are [ember-vite-codemod](https://github.com/mainmatter/ember-vite-codemod)'s doing. ember-vite-codemod allows you to migrate an Ember app to a Vite app with just one command.